packages/colony-starter-vue/src/components/Me.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <p v-if="loading">loading...</p>
+    <div v-if="!loading">
+      <p>Wallet Address: {{colonyClient.adapter.wallet.address}}</p>
+      <p>Network Address: {{colonyClient.networkClient.contract.address}}</p>
+      <p>Colony Address: {{colonyClient.contract.address}}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Me',
```

Comment:
I'll consolidate them.  
I agree, `Me` is not a proper name considering the elements it contains.  

The reason to separate between `views` and `components` is to follow the same pattern react does of `containers` and components`.  
Components in `components` try to not have `data` but instead receive them from `props`, and components in `views` use the former components and pass to them the appropriate props.  
This makes components much easier to debug.

Reply:
Ok, that separation makes sense and seems to be popular with view projects. I was just used to the react structure. 👍 

Let's just rename the `Me` component. Would it make sense to rename it to `Home` so that the `Home` component provides logic for the `Home` view?

Reply:
Makes sense. This is done.